app/client/components/helpCentre/HelpTopicBox.tsx

```diff
@@ -57,7 +57,7 @@ const linkListItemStyle = css`
   position: relative;
 `;
 
-const linkAnchorStyle = css`
+export const linkAnchorStyle = css`
```

Comment:
Because `HelpTopicBox` and `HelpCentreFaq` components don't depend on each other perhaps it's best to move these css declarations into a separate file that both components import. We could start a `helpCentreStyles` file where we put all the styles we need to share across components.

Reply:
Sure, I can start a separate file for shared Help Centre styles starting with these two variables 👍